adcc/testdata/qchem/__init__.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
```

Comment:
TODO to @mfherbst: Merge this with the get testdata script and upload the file to the adc-connect.org server.